packages/bygger/src/translations/TranslationsByFormPage.jsx

```diff
@@ -29,20 +30,45 @@ const useStyles = makeStyles({
   },
 });
 
-const TranslationsByFormPage = ({ deleteTranslation, saveTranslation, form, languageCode, projectURL }) => {
```

Comment:
Hva skjer dersom man ikke lagrer skjemaet før man trykker på "Oversettelser"-knappen? Må vi tvinge en lagring?

Reply:
Per nå vil man miste endringene sine. Slik jeg ser det har vi følgende muligheter:

1. Vis en advarsel før man forlater EditFormPage med endringer som ikke er lagret.
2. Flytt TranslationsByFormPage inn under FormPage (/form/:formPath/translations/:langCode

Valg to kunne jo være fint, siden vi da slipper å laste form hver gang vi hopper mellom oversettelse og rediger skjema.
Samtidig, så fører det til at vi potensielt kan lage oversettelser for komponenter som ikke er lagret, eller at vi må lagre form og oversettelser på en gang. Det blir en del omskriving, og kanskje ikke helt oversiktlig for bruker.

Jeg stemmer for alternativ 1.